dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/FATSuite.java

```diff
@@ -44,4 +47,12 @@
 })
 
 public class FATSuite {
+
+    public static final String BETA_ID = MicroProfileActions.MP61_ID + "_BETA";
+
+    public static RepeatTests aboveMP50Repeats(String serverName) {
+        return MicroProfileActions
+                        .repeat(serverName, TelemetryActions.MP50_MPTEL11, MicroProfileActions.MP60, MicroProfileActions.MP61)
```

Comment:
Not a good idea to mix TelemetryActions and MicroProfileActions... but I suspect you get away with it in this case. TelemetryActions does not contain all the features that MicroProfileActions does (and vice versa) ... so it potentially possible that a feature would not be correctly removed from the server.xml.